x-pack/platform/plugins/shared/streams_app/public/components/data_management/schema_editor/hooks/use_schema_fields.ts

```diff
@@ -120,6 +120,7 @@ export const useSchemaFields = ({
               ingest: {
                 ...definition.stream.ingest,
                 wired: {
+                  ...definition.stream.ingest.wired,
```

Comment:
Shouldn't this change be also applied when updating the definition in the enrichment part? Specifically, I refer to this point of the update request: https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_enrichment/hooks/use_definition.ts#L145

Reply:
Yeah I'll fix the type in the state management PR after rebasing this changes